sysutils/starship/Portfile

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        starship starship 0.15.0 v
+categories          sysutils
+platforms           darwin
+maintainers         {l2dy @l2dy} openmaintainer
+license             ISC
+
+description         a minimal, blazing fast, and extremely customizable prompt for any shell
+
+long_description    Starship is ${description}.
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  4fb6c4f6cd26dfecc074dfc08b32fd95d55114cf \
+                    sha256  fe321171cb34219e17c73ce072c141929575666c103047eb1d0f861ca54ee5a4 \
+                    size    4192407
+
+# For crate:openssl-sys
+depends_build       port:pkgconfig
+depends_lib         path:lib/libssl.dylib:openssl
```

Comment:
When using port groups, it is best practice to use `depends_build-append` and `depends_lib-append` instead so you don't overwrite current or future portgroup dependencies.

Reply:
Thanks for the review! But I'll rely on the workaround provided in `cargo_fetch 1.0` for now, to make the code look cleaner.